.github/scripts/common.sh

```diff
@@ -24,6 +24,7 @@ ci_run_jl_test() {
 
     cd $JULIA_PATH
     export JULIA_CPU_THREADS=$threads
```

Comment:
We can possibly just remove this. I think I probably have used the wrong env vars.